autoload/gina/command/blame.vim

```diff
@@ -180,6 +177,24 @@ function! s:BufReadCmd() abort
   setlocal filetype=gina-blame
 endfunction
 
+function! s:blame_buffer_names() abort
+  let bufname = bufname('%')
+  let scheme = gina#core#buffer#param(bufname, 'scheme')
+  let alternate = substitute(
+        \ bufname,
+        \ ':' . scheme . '\>',
+        \ ':' . (scheme ==# 'blame' ? 'show' : 'blame'),
+        \ ''
+        \)
+  return {'current': bufname, 'alternate': alternate}
+endfunction
+
+function! s:exit_from_entire_blame() abort
+  let buffers = s:blame_buffer_names()
+  exe 'silent! bwipeout '.buffers.alternate
+  exe 'silent! bwipeout '.buffers.current
```

Comment:
1. Use full command names (e.g. `execute`)
2. Do not use `.` while `execute` automatically concatenate the attributes with spaces (`execute 'silent! bwipeout' buffer.alternate` is enough)
3. I prefer to use `quit` or `close` instead of using `bwipeout` directly
4. What's happen if there are only two buffer exists? Does Vim quit or a new empty buffer automatically open?